.github/scripts/transform-message.ts

```diff
@@ -18,244 +19,163 @@ interface GitHubAction
-          discussion?.title
-        }`,
-        content: [
-          [
-            { tag: "text", text: "链接：" },
-            {
-              tag: "a",
-              text: discussion?.html_url,
-              href: discussion?.html_url,
-            },
-          ],
-          [
-            { tag: "text", text: "作者：" },
-            {
-              tag: "a",
-              text: discussion?.user?.login,
-              href: discussion?.user?.html_url,
-            },
-          ],
-          [
-            {
-              tag: "text",
-              text: `分类：${discussion?.category.name || ""}`,
-            },
-          ],
-          [
-            {
-              tag: "text",
-              text: `标签：${
-                discussion?.labels?.map(({ name }) => name).join(", ") || ""
-              }`,
-            },
-          ],
-          [
-            { tag: "text", text: "描述：" },
-            { tag: "text", text: discussion?.body },
-          ],
-        ],
-      }
-    : event_name === "issue_comment" || event_name === "discussion_comment"
-    ? {
-        title: `GitHub 帖子评论：${issue?.title || discussion?.title}`,
-        content: [
-          [
-            { tag: "text", text: "链接：" },
-            {
-              tag: "a",
-              text: comment?.html_url,
-              href: comment?.html_url,
-            },
-          ],
-          [
-            { tag: "text", text: "作者：" },
-            {
-              tag: "a",
-              text: comment?.user?.login,
-              href: comment?.user?.html_url,
-            },
-          ],
-          [
-            { tag: "text", text: "描述：" },
-            { tag: "text", text: comment?.body },
-          ],
-        ],
-      }
-    : event_name === "release" && action === "published"
-    ? {
-        title: `GitHub Release 发布：${release?.name || release?.tag_name}`,
-        content: [
-          [
-            { tag: "text", text: "链接：" },
-            {
-              tag: "a",
-              text: release?.html_url,
-              href: release?.html_url,
-            },
-          ],
-          [
-            { tag: "text", text: "作者：" },
-            {
-              tag: "a",
-              text: release?.author?.login,
-              href: release?.author?.html_url,
-            },
-          ],
-          [
-            { tag: "text", text: "描述：" },
-            { tag: "text", text: release?.body },
-          ],
-        ],
-      }
-    : null;
+const createLink = (href: string, text = href) => ({ tag: "a", href, text });
+
+const createText = (text: string) => ({ tag: "text", text });
+
+// 新增辅助函数
+const createUserLink = (user?: { login: string; html_url: string }) => 
+  user ? createLink(user.html_url, user.login) : createText("无");
+
+const createContentItem = (label: string, value?: string | { tag: string; text: string }) => [
+  createText(label),
+  typeof value === 'string' ? createText(value || "无") : value || createText("无")
+];
```

Comment:
```suggestion
const createContentItem = (label: string, value?: string | { tag: string; text: string }) => [
  createText(label),
  typeof value === 'string' ? createText(value || "无") : value || createText("无")
] as [object, object];
```
下面用的时候的 `any` 断言就可以删了。